### PR TITLE
fix: Fix options in notebooks

### LINKED ISSUE #1951: Maps: fix broken customization options in Notebooks

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #1951 

Alright this fixes the options in notebooks.

### CODE REVIEW

File: src/visualization/types/Map/GeoOptions.tsx [hunk 1]
```diff
@@ -36,6 +39,27 @@ export enum MapType {
 
 export const GeoOptions: FC<Props> = ({properties, update, results}) => {
   const {useS2CellID} = properties
+
+  if (!properties.layers.length) {
```
Comment (TCL735) [inline src/visualization/types/Map/GeoOptions.tsx:43]: `!properties?.layers?.length` is probably better unless you can guarantee `properties.layers` always exists here

File: src/visualization/types/Map/GeoOptions.tsx [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import React, {CSSProperties, FC} from 'react'
+import _ from 'lodash'
```
Comment (genehynson) [inline src/visualization/types/Map/GeoOptions.tsx:2]: please don't import all of lodash - just import the functions you need 🙏

**Comment by genehynson** [comment]
do we have any tests for rendering maps visualization in notebooks?

File: src/visualization/types/Map/GeoOptions.tsx (hunk 2 above)
Comment (zoesteinkamp) [reply src/visualization/types/Map/GeoOptions.tsx:2]: got it!

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 51 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1958)

**Review by Palakp41 — CHANGES_REQUESTED** [review]
I think instead of updating properties and adding colors in GeoOptions, seems a bit weird. What if instead of checking if layers is empty here, what if we update layers to not be an empty array here: https://github.com/influxdata/ui/blob/master/src/visualization/types/Map/properties.ts#L42?

We can use the default values like this: 
```
    {
      type: 'pointMap',
      colorDimension: {label: 'Value'},
      colorField: '_value',
      colors: DEFAULT_THRESHOLDS_GEO_COLORS,
      isClustered: false,
      tooltipColumns: [],
    }
  ```

**Review by Palakp41 — APPROVED** [review]
Looks good to me!

**Pull request merged**